mobile/library/swift/EngineBuilder.swift

```diff
@@ -612,6 +616,21 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Adds a CDS layer to the configuration.
+  ///
+  /// - parameter cdsResourcesLocator: The xdstp resource URI for fetching clusters.
+  /// - parameter timeoutSeconds:      The timeout in seconds.
```

Comment:
It would be useful to explain what a zero-second timeout does and what an empty locator does.